setup.py

```diff
@@ -96,4 +96,32 @@
         ],
     },
     install_requires=requirements,
+    # these links will be shown in the left panel on PyPI
+    project_urls={
+        "documentation": "https://xcube.readthedocs.io/en/latest/",
+        "source": "https://github.com/xcube-dev/xcube",
+        "download": "https://pypi.org/project/xcube-core/#files",
+        "tracker": "https://github.com/xcube-dev/xcube/issues",
+        "release notes": "https://github.com/xcube-dev/xcube/releases"
+    },
```

Comment:
Missing changelog and repository here. Look, in zappend's `pyproject.toml` we have 
```
[project.urls]
Documentation = "https://bcdev.github.io/zappend/"
Issues = "https://github.com/bcdev/zappend/issues"
Changelog = "https://github.com/bcdev/zappend/blob/main/CHANGES.md"
Repository = "https://github.com/bcdev/zappend"
```

Reply:
Repository appears as 'Homepage' on [zappend PyPI](https://pypi.org/project/zappend/). PyPI recognizes "source" as GitHub and tags it with the GitHub sign on the [xcube-core TestPyPI](https://test.pypi.org/project/xcube/1.6.0.dev12/), which I like more. I'd prefer "source", except you have strong objections. 

I can add Changelog. However, this is very similar to "release notes" except that the changes of the current development are missing. Should we add both? 

Note that I took [numpy PyPI](https://pypi.org/project/numpy/) as inspiration. However, I just noticed that there links are with capital letters. Probably when using `pyproject.toml`, it modifies the field names to capitalized words which is not done with `setup.py`. I will change that accordingly.

Reply:
> I'd prefer "source", except you have strong objections.

I agree.

> I can add Changelog. However, this is very similar to "release notes" except that the changes of the current development are missing. Should we add both?

Yes please.

> I will change that accordingly.

Yes, if that works.

Reply:
All changes are applied, which can be viewed here on [TestPyPI xcube-core](https://test.pypi.org/project/xcube/1.6.0.dev14/)